contribute/git-commit-message.md

```diff
@@ -0,0 +1,65 @@
+# Commit Message Guidelines for OpenEBS Projects
+
+This document borrows concepts, conventions, and text mainly from the following sources, extending them in order to provide a sensible guideline for writing commit messages for OpenEBS projects.
+- Tim Pope's [article](https://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html) on readable commit messages
+- Thanks to @stephenparish https://gist.github.com/stephenparish/9941e89d80e2bc58a153
+- Thanks to @abravalheri https://gist.github.com/abravalheri/34aeb7b18d61392251a2
+
+These conventions are aimed at tools to automatically generate useful documentation, or by developers during debugging process.
+
+## Proposed Commit Message Format
+
+Any line of the commit message cannot be longer than 80 characters! This allows the message to be easier to read on github as well as in various git tools.
+
+```
+<type>(<scope>): <subject> <meta>
+<BLANK LINE>
+<body>
+<BLANK LINE>
+<footer>
+```
+
+### Allowed `<type>`
+* **feat**: A new feature
+* **fix**: A bug fix
+* **docs**: Documentation only changes
+* **style**: Changes that do not affect the meaning of the code 
+  (white-space, formatting, missing semi-colons, etc)
+* **refactor**: A code change that neither fixes a bug nor adds a feature
+* **perf**: A code change that improves performance
+* **test**: Adding missing tests
+* **chore**: Changes to the build process or auxiliary tools and libraries
+   such as documentation generation
```

Comment:
These are all we have on mayastor: 'build', 'chore', 'ci', 'docs', 'feat', 'fix', 'perf', 'refactor', 'revert', 'style', 'test', 'example', 'security'

Reply:
How about, I redirect folks to this? https://www.conventionalcommits.org/en/v1.0.0/#summary

Reply:
Added all of them.